api/AudioContext.json

```diff
@@ -333,7 +333,7 @@
           "spec_url": "https://webaudio.github.io/web-audio-api/#dom-audiocontext-createmediaelementsource",
           "support": {
             "chrome": {
-              "version_added": "14"
+              "version_added": "15"
```

Comment:
https://mdn-bcd-collector.gooborg.com/tests/api/AudioContext/createMediaElementSource is simple enough that I trust this.